values/fetch_historical_values.py

```diff
@@ -0,0 +1,22 @@
+import csv
+
+class HistoricalValueFetcher:
+    def __init__(self):
+        list_of_values = []
+        path = "Github/Crazy_tr8/historical_data/XBTUSD_Q3.csv"
+        with open(path, 'r') as file_csv:
+            csv_reader = csv.reader(file_csv)
+            for self.row in csv_reader:
+                list_of_values.append(self.row)
+
+    def get_historical_price(self):
+        try:
+            return self.row[1]
```

Comment:
I think this is wrong you will always return 1 item form historical values while we are always want to get next